pkg/cmd/grafana-server/wireexts_enterprise.go

```diff
@@ -0,0 +1,18 @@
+// +build wireinject,enterprise
+
+package main
+
+import (
+	"github.com/google/wire"
+	"github.com/grafana/grafana/pkg/extensions"
+	"github.com/grafana/grafana/pkg/extensions/licensing"
+	"github.com/grafana/grafana/pkg/models"
+)
+
+var wireExtsSet = wire.NewSet(
+	wireSet,
+	licensing.ProvideLicensing,
+	licensing.ProvideLicenseTokenService,
+	wire.Bind(new(models.Licensing), new(*licensing.LicenseTokenService)),
+	extensions.ProvideAPI,
+)
```

Comment:
Can this be in enterprise repo?

Reply:
@torkelo yeah, @xlson asked about the same and I think it should be in enterprise repo and symlinked in.